bin/floydsauto

```diff
@@ -88,8 +89,18 @@ if __name__ == "__main__":
         camera = FLOYDS_CAMERAS[option.site]
     else:
         camera = option.camera
-    fpacked_files = glob.glob(os.path.join(option.input_directory, option.site, camera,
-                                           option.day_obs, 'raw', '*.fits.fz'))
+
+    raw_data_path = glob.glob(os.path.join(option.input_directory, option.site, camera,
+                                           option.day_obs, 'raw', ''))
+    if not os.path.exists(option.input_directory):
+        os.makedirs(raw_data_path)
+        url = os.getenv('API_ROOT', '') + 'frames/?siteid={site}&instrume={instrument}&basename={day_obs}'.format(site=option.site, instrument=camera, day_obs=option.day_obs),
```

Comment:
Looks like you're substituting day_obs into the basename parameter here. 

You probably want something like 

`frames/?siteid={site}&instrume={instrument}&day_obs={day_obs}`, since you are relying on the archive API to give you a list of frames without knowing their basenames beforehand.